schemas/request-metadata-property.json

```diff
@@ -0,0 +1,29 @@
+{
+    "type": "object",
+    "additionalProperties": false,
```

Comment:
We should allow for extra properties.  Most of the time this won't matter, but it could make it easier to integrate with.  for instance.

```
metadataProperty.some_thing_only_a_client_would_care_about = "hello";
// some time later
metadataPropertyRouteHelper.put(metadataProperty)
```

In that case it would still work.  Granted, that has its own problem (They could misspell "immutable" and we would ignore its values) but that is the methodology we picked at the beginning.